certbot/cert_manager.py

```diff
@@ -73,6 +74,7 @@ def certificates(config):
     for renewal_file in storage.renewal_conf_files(config):
         try:
             renewal_candidate = storage.RenewableCert(renewal_file, config)
+            crypto_util.verify_renewable_cert(renewal_candidate)
```

Comment:
Thanks for the suggestion, I've added more descriptive error messaging/exception handling.